build/Packages.props

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
```

Comment:
ooh i like that! 👍👍

Reply:
Holy verbosity!  (Though a useful change)